packages/main/test/samples/Input.sample.html

```diff
@@ -141,7 +141,7 @@ <h3>Input with Value State</h3>
 <section>
 	<h3>Input with Suggestions and Value State message</h3>
 	<div class="snippet">
-		<ui5-input class="samples-margin samples-responsive-margin-bottom" id="value-state-suggestions" class="samples-responsive-padding-bottom" placeholder="Start typing country name" show-suggestions value-state="Warning">
+		<ui5-input class="samples-margin samples-responsive-margin-bottom" id="value-state-suggestions" placeholder="Start typing country name" show-suggestions value-state="Warning">
```

Comment:
These classes are required, otherwise the result is this:

![image](https://user-images.githubusercontent.com/16850339/133049851-e12c6c04-19ed-42c2-a4ef-8ecec69622dc.png)